src/skills-builder/skills-builder-modal/select-preferences/JobTitleSelect.jsx

```diff
@@ -28,17 +29,20 @@ const JobTitleSelect = () => {
         <FormattedMessage {...messages.jobTitlePrompt} />
       </h4>
       <InstantSearch searchClient={searchClient} indexName={getConfig().ALGOLIA_JOBS_INDEX_NAME}>
-        <JobTitleInstantSearch onSelected={handleCurrentJobTitleSelect} />
+        <JobTitleInstantSearch
+          onSelected={handleCurrentJobTitleSelect}
+          placeholder={currentJobTitle}
+        />
       </InstantSearch>
       <Form.Group>
         <Form.CheckboxSet
           name="other-occupations"
           onChange={handleCheckboxChange}
         >
-          <Form.Checkbox value="student">
+          <Form.Checkbox value="Student">
             <FormattedMessage {...messages.studentCheckboxPrompt} />
```

Comment:
Should we have updated this line (and line 46) with the rest of the i18n changes?